vic_run/src/calc_surf_energy_bal.c

```diff
@@ -192,16 +192,12 @@ calc_surf_energy_bal(double             Le,
         while (soil_con->Zsum_node[i] < soil_con->depth[0]) {
             i++;
         }
-        T1_old =
-            energy->T[i -
-                      1] +
-            (energy->T[i] -
-             energy->T[i -
-                       1]) *
-            (soil_con->depth[0] -
-             soil_con->Zsum_node[i -
-                                 1]) /
-            (soil_con->Zsum_node[i] - soil_con->Zsum_node[i - 1]);                                                                                                        // i should be greater than 0, else code would have aborted in initialize_model_state()
+        T1_old = energy->T[i - 1] +
+                 (energy->T[i] - energy->T[i - 1]) *
+                 (soil_con->depth[0] -
+                  soil_con->Zsum_node[i -
+                                      1]) /
```

Comment:
done.  (I've been fixing these as I go through the code but this is a uncrustify thing.